platform-sdk/swirlds-config-impl/src/main/java/com/swirlds/config/impl/internal/ConfigurationImpl.java

```diff
@@ -90,6 +95,9 @@ public <T> T getValue(
     @Override
     public String getValue(@NonNull final String propertyName) {
         ArgumentUtils.throwArgBlank(propertyName, "propertyName");
+        if (propertiesService.isListProperty(propertyName)) {
```

Comment:
I ask myself if that exception should be at this palce or better in the propertyService